modules/dag-l1/src/main/scala/org/tessellation/dag/l1/StateChannel.scala

```diff
@@ -173,7 +182,11 @@ class StateChannel[F[_]: Async: KryoSerializer: SecurityProvider: Random](
     .awakeEvery(10.seconds)
     .evalMap(_ => services.l0.pullGlobalSnapshots)
     .flatMap(snapshots => Stream.fromIterator(snapshots.iterator, 1))
-    .evalMap(snapshot => logger.info(s"Pulled following global snapshot: $snapshot"))
+    .evalMap { snapshot =>
+      snapshot.hashF.flatMap { hash =>
+        logger.info(s"Pulled following global snapshot: ${snapshot.ordinal.show} ${hash.show}")
+      }
+    }
```

Comment:
Not necessary as it will change with the download/redownload in L1 PR

Reply:
Ok, so you'll remove that